docs/index.md

```diff
@@ -154,7 +154,7 @@ There are a few things that can be checked if you review a pull request against
 * Does it have failing tests? Add the `tests-fail` label
 * Does it drop support for a specific Operating system or a major Puppet version? Add the `backwards-incompatible` label
 * Are new parameters introduced? They must have datatypes
-* Are facts used? They should only be accessed via `$facts[]` or [fact()](https://github.com/puppetlabs/puppetlabs-stdlib#fact) from stdlib, but not topscope variables
+* Are facts used? They should only be accessed via `$facts[]` or [fact()](https://github.com/puppetlabs/puppetlabs-stdlib#facts) from stdlib, but not topscope variables
```

Comment:
The link was for the `fact()` function, not for the facts provided by the stdlib module. The new link is https://github.com/puppetlabs/puppetlabs-stdlib/blob/master/REFERENCE.md#fact

Reply:
done